docu/files/Multi-tabEditor.html.md

```diff
@@ -0,0 +1,96 @@
+---
+title: 'The multi-tab editor component in the Builder'
+service: 'Builder SDK'
+area: 'Core'
+type: Tutorial
+order: 4
+---
+
+The multi-tab editor is a custom Angular directive in **builder_editors** with a dependency to the **builder**.
+
+### Configure the multi-tab editor
+
+As a Builder module developer, you need to complete a few simple tasks:
+
+* Include the https://builder.yaas.io/public/js/builder_editors_v3.js file. Add **builder_editors** to your module.
+* The directive expects data formatted as in this example:
+
+```js
+    /*simple string with localized property label */
+    $scope.localizedLabel = 'Demo Label';
+    
+    $scope.configuration = {
+    'en': {'default': true, 'required': true},
+    'de': {'default': false, 'required': false}
+    };
+
+```
+Specify the data that you would like to display in the element. The multi-tab editor also accepts localized data formatted as in this example:
```

Comment:
Specify the data to display in the element. The multi-tab editor also accepts localized data formatted as shown in this example: